game-core/src/main/java/org/triplea/util/ExitStatus.java

```diff
@@ -14,10 +14,10 @@
   /** The process exited due to a failure (1). */
   FAILURE(1);
 
+  private static final Collection<Runnable> exitActions = new HashSet<>();
   private final int status;
-  private final List<Runnable> exitActions = new ArrayList<>();
 
-  public void addExitAction(final Runnable runnable) {
+  public static void addExitAction(final Runnable runnable) {
     exitActions.add(runnable);
   }
```

Comment:
Ok now there's no difference to https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Runtime.html#addShutdownHook(java.lang.Thread)
Thoghts about removing this code?

Reply:
I didn't realize that API existed, good to learn about. It looks like we should indeed use that and remove our custom listener implementation.